src/controllers/v1/entityTypes.js

```diff
@@ -57,31 +58,134 @@ module.exports = class EntityTypes extends Abstract {
   ]
 */
 
-  /**
-   * List all the entity types.
-   * @method
-   * @name list 
-   * @returns {JSON} - List of all entity types.
-   */
-
-  async bulkCreate(req) {
-    return new Promise(async (resolve, reject) => {
-      try {
-
-        let result = {
-          status: 200,
-          message: "OK"
-        }
-
-        return resolve(result)
-
-      } catch (error) {
-        return reject({
-          status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
-          message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
-          errorObject: error
-        })
-      }
-    })
-  }
-}
\ No newline at end of file
+	/**
+	 * List all the entity types.
+	 * @method
+	 * @name list
+	 * @returns {JSON} - List of all entity types.
+	 */
+
+	list() {
+		return new Promise(async (resolve, reject) => {
+			try {
+				let result = await entityTypesHelper.list('all', { name: 1 })
+
+				return resolve({
```

Comment:
@MallanagoudaB let the message also pass from the helper file and return only result here